lib/ex_pix_brcode/changesets.ex

```diff
@@ -57,4 +58,25 @@ defmodule ExPixBRCode.Changesets do
   end
 
   def valid_document?(_document, _field), do: false
+
+  @doc """
+  'chave' and 'infoAdicional' validation.
+
+  These fields have the following rules:
+    - 
+  """
+  def validate_chave_and_info_adicional_length(%{valid?: false} = c, _, _, _), do: c
```

Comment:
oq acha de mudar tudo para ingles e no doc especifica que por exemplo, key é o campo chave e add_info é o campo infoAdicional?
```suggestion
  def validate_key_and_additional_info_length(%{valid?: false} = c, _, _, _), do: c
```

Reply:
Fiquei em dúvida aqui por ser uma mistura de inglês e português...

O campo chama realmente `info_adicional` e chave. Daí pensei que indica melhor ONDE é usado deixando assim.

Reply:
Entendi. Eu ainda acho confuso essa mistura e como tem a documentação da função, nela poderia testar contida a referencia para o campo `chave` e para o campo `info_adicional`, mas enfim, é algo mais 💅🏽 mesmo